test/production/app-dir/import-trace-dynamic-eval/import-trace-dynamic-eval.test.ts

```diff
@@ -0,0 +1,27 @@
+import { nextBuild } from 'next-test-utils'
+
+describe('import trace dynamic eval', () => {
+  it("should show the user's import trace", async () => {
+    const output = await nextBuild(__dirname, [], {
```

Comment:
It should be in e2e tests to cover both dev and prod, and we can leverage the utils to avoid calling nextBuild manually. Using e2e can also benefit for turbopack.

This is still an integration test, with e2e test we're separating the env into a new temporary folder and run every there, including dependencies. That's why I was also asking to have a simpler dependency instead of ajv.

There're a few places are checking `Dynamic Code Evaluation` in existing test already, this is duplicated as `test/production/edge-safe-dynamic-code/index.test.ts`, we could just check module trace there.